crates/router/src/core/payment_methods/cards.rs

```diff
@@ -1324,7 +1324,15 @@ impl BasiliskCardSupport {
             .change_context(errors::ApiErrorResponse::InternalServerError)
             .attach_printable("Wrapped value2 construction failed when saving card to locker")?;
 
-        vault::create_tokenize(state, value1, Some(value2), payment_token.to_string()).await?;
+        let lookup_key =
+            vault::create_tokenize(state, value1, Some(value2), payment_token.to_string()).await?;
+        vault::add_tokenize_data_task(
```

Comment:
Can the function name related to the behavior.

Reply:
tokenize - tokenized

Reply:
add_delete_tokenized_data_task